src/components/Solution.js

```diff
@@ -0,0 +1,16 @@
+import Letter from "./Letter";
+import React, { Component } from 'react';
+
+
+class Solution extends Component {
+    render() {
+        const arr = ["_ ", "_ ", "_ ", "_ "]
+        return (
+            <div id="solution">
+                <div>{this.props.word.split('').map((letter, index)=> this.props.letterStatus[letter] ? <Letter key={index} letter={letter} /> : <Letter key={index} letter="-" />)} 
+                </div>
```

Comment:
It can be better like this: `<Letter key={index} letter={this.props.letterStatus[letter] ? letter : "-"}/>`